service/controller/app/v1/resource/chartcrd/resource.go

```diff
@@ -0,0 +1,53 @@
+package chartcrd
+
+import (
+	"github.com/giantswarm/apiextensions/pkg/clientset/versioned"
+	"github.com/giantswarm/kubeconfig"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"k8s.io/client-go/kubernetes"
+)
+
+const (
+	Name = "chartcrdv1"
+)
+
+// Config represents the configuration used to create a new chartstatus resource.
+type Config struct {
+	G8sClient  versioned.Interface
+	K8sClient  kubernetes.Interface
+	KubeConfig kubeconfig.Interface
+	Logger     micrologger.Logger
+
+	WatchNamespace string
+}
+
+// Resource implements the chartstatus resource.
```

Comment:
I am wondering right now why the resource is installing the CRD and where. We could maybe have some more useful godoc and describe in which part of the infrastructure this resource acts in and why it does so. Just like drawing a little picture. What do you think about that?

Reply:
Sorry, we can keep go in https://github.com/giantswarm/app-operator/pull/33#discussion_r254644232